numba/np/arrayobj.py

```diff
@@ -1563,6 +1563,22 @@ def array_T(context, builder, typ, value):
     return impl_ret_borrowed(context, builder, typ, res)
 
 
+@overload(np.logspace)
+def numpy_logspace(start, stop, num=50):
+    if not isinstance(start, types.Number):
+        raise errors.TypingError('The first argument "start" must be a number')
+    if not isinstance(stop, types.Number):
+        raise errors.TypingError('The second argument "end" must be a number')
```

Comment:
```suggestion
    if not isinstance(stop, types.Number):
        raise errors.TypingError('The second argument "stop" must be a number')
```
The argument is called stop?

Reply:
Yes, the argument is called stop, which is reference to implementation in Numpy.